x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/action_clone/clone_button.tsx

```diff
@@ -353,7 +353,7 @@ interface CloneActionProps {
  * Replace with {@link getCloneAction} as soon as all the actions are refactored
  * to support EuiContext with a valid DOM structure without nested buttons.
  */
-export const CloneAction: FC<CloneActionProps> = ({ createAnalyticsForm, item }) => {
+export const CloneButton: FC<CloneButtonProps> = ({ createAnalyticsForm, item }) => {
```

Comment:
You probably overlooked the comment on top of this component 🙂 Consider using `getCloneAction` instead

Reply:
> Note that this PR doesn't fix the nested buttons issue (#63455) yet. For that we need EUI issue #70383 to be in Kibana which will arrive with EUI v26.3.0 via #70243. So there will be one follow up to that which will focus on getting rid of the nested button structure.

We cannot migrate all buttons to this pattern yet, it depends on another upcoming EUI update.